lib/content-services/src/lib/search/services/search-facet-filters.service.ts

```diff
@@ -335,6 +342,25 @@ export class SearchFacetFiltersService implements OnDestroy {
         };
     }
 
+    private loadCategoryNames(bucketList: FacetFieldBucket[]) {
+        bucketList.forEach((item) => {
```

Comment:
needs verification, but I suspect these will run in parallel, and may exit the loop on the first observable error; probably needs error handling for the subscribes

Reply:
Added error handling, take a look please